tests/containers/validate_btrfs.pm

```diff
@@ -104,7 +104,9 @@ sub run {
     my $docker    = containers::runtime->new(runtime => 'docker');
     my $btrfs_dev = '/var/lib/docker';
     my ($untested_images, $released_images) = get_suse_container_urls();
-    _sanity_test_btrfs($docker, $btrfs_dev, $released_images->[0]);
+    my $images_to_test = check_var('CONTAINERS_UNTESTED_IMAGES', '1') ? $untested_images : $released_images;
+
+    _sanity_test_btrfs($docker, $btrfs_dev, $images_to_test->[0]);
```

Comment:
If `$images_to_test` is an array, why only test the first URL?

Reply:
When the module implemented, it was one image in the list. i keep that as it is. 
Now this is scheduled on host-based tests. which means that it doesnt have to run against various images. if it works for one it should be fine

Reply:
Yes, we need to find a trade-off here... We already run `podman_image.pm` for a bunch of released images and it's time consuming. So, testing single image is fine for me, and it doesn't have to be a sle image, it could be some random image...